pkg/chartutil/create.go

```diff
@@ -308,6 +309,9 @@ spec:
       {{- end }}
       labels:
         {{- include "<CHARTNAME>.selectorLabels" . | nindent 8 }}
+	{{- with .Values.podLabels }}
```

Comment:
Line 312 is tab-indented, while others are space-indented. Should they be consistent?

Reply:
oh sorry

Reply:
https://github.com/helm/helm/pull/12457